README.md

```diff
@@ -142,11 +118,82 @@ pytest --runslow
 
 ## Usage
 
-*Updates still to come*.
+The documentation for Firecrown is still under development.
+It is [available on readthedocs](https://firecrown.readthedocs.io/).
+
+> Warning
+>
+> The documentation currently online is out-of-date. We are working on updating it.
+> The source for the documentation is in this repository, under the `docs/` subdirectory.
+> If you have the repository cloned, and one of the conda environments described below active,
+> the following steps will build the documentation into `docs/_build/html`:
+>
+>     # Installation of the required Python packages to build the documentation only needs to
+>     # be done once per environment. Make sure you are in an active environment before doing
+>     # this installation.
+>     conda install -c conda-forge sphinx sphinx-autodoc-typehints sphinx_rtd_theme
+>
+>     cd docs/
+>     make html
+
+There are examples in the [examples folder](https://github.com/LSSTDESC/firecrown/examples)
+that show some use of Firecrown with both Cobaya and CosmoSIS.
+
+## Contributing
+
+Contributions to Firecrown are welcome.
+
+For any contribution, please start by [opening an issue](https://github.com/LSSTDESC/firecrown/issues),
+and using the GitHub interface to create a branch for that issue.
```

Comment:
People will only be able to create a branch if they have write access (i.e. if they are on the 22-member `firecrown-devs` team).  If they are not, they would have to make their own fork.  i suggest expanding this to mention that option, and tell people how they can request write access.  For example, something like "To create a branch you will need write access; if you don't have write access, you can request it by asking to join the [@LSSTDESC/firecrown-devs](https://github.com/orgs/LSSTDESC/teams/firecrown-dev/members") team."